src/DependencyRewriterPlugin.php

```diff
@@ -71,6 +70,16 @@ public function activate(Composer $composer, IOInterface $io)
         $this->output(sprintf('<info>Activating %s</info>', __CLASS__), IOInterface::DEBUG);
     }
 
+    public function deactivate(Composer $composer, IOInterface $io) : void
+    {
+        // Nothing to do here, as all features are provided through event listeners
+    }
+
+    public function uninstall(Composer $composer, IOInterface $io) : void
+    {
+        // Nothing to do here, as all features are provided through event listeners
```

Comment:
Actually, we are not even handling `uninstall` events with this plugin, thus we should not say assume that `uninstall` is handled by any listener.

Reply:
comment updated